### DISCUSSION TITLE
Single command to control all services

### DISCUSSION THREAD

**Discussion by layer3-fl** [modoboa/modoboa#3211] [Help]
Good morning,

I have successfully installed Modoboa on a fresh install of Debian 12 on my hosted dedicated server and have been up and running for a few days with several domains hosted. So far everything has been running pretty smoothly and I want to thank you guys for putting out a great product.

Name 	Installed version 	Last version 	Description
Modoboa 	2.2.4 	2.2.3 	The core part of Modoboa

Having migrated from Zimbra Community which EOL'd last December, I was looking for the existence of a single command to control all related services rather than dealing with each one individually after making changes to config files, system settings, etc. For example Zimbra has the zmcontrol command which can start/stop/restart related services.

[zimbra@server ~]$ zmcontrol start
Host server.example.com
        Starting zmconfigd...Done.
        Starting logger...Done.
        Starting mailbox...Done.
        Starting memcached...Done.
        Starting proxy...Done.
        Starting amavis...Done.
        Starting antispam...Done.
        Starting antivirus...Done.
        Starting opendkim...Done.
        Starting snmp...Done.
        Starting spell...Done.
        Starting mta...Done.
        Starting stats...Done.
        Starting service webapp...Done.
        Starting zimbra webapp...Done.
        Starting zimbraAdmin webapp...Done.
        Starting zimlet webapp...Done.

I have searched Documentation and Issues/discussions here and can not find or am missing the answer. 

Thanks for any information you can share.

-Matto

**Comment by tonioo** [comment]
There is no such command in Modoboa at the moment.

**Reply by layer3-fl** [reply]
Thanks for the response!
